src/java/org/apache/cassandra/service/ActiveRepairService.java

```diff
@@ -20,10 +20,25 @@
 import java.io.IOException;
 import java.lang.management.ManagementFactory;
 import java.net.UnknownHostException;
-import java.util.*;
-import java.util.concurrent.*;
+import java.util.ArrayList;
```

Comment:
Import fighting still happening?

Reply:
Yes, I'll remove it on commit when it's squashed.